tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "outDir": "lib/",
     "strict": false,
     "target": "es2018",
+    "declaration": true,
   },
-  "include": ["next-env.d.ts", "src/*.ts", "src/*.tsx"],
+  "include": ["next-env.d.ts", "src/*.ts", "src/*.tsx", "src/**/*.ts", "src/**/*.tsx"],
```

Comment:
why this addition? doesn't `"src/*.ts, "src/*.tsx"` covers everything?

Reply:
It was not working for subfolders on my Windows VM for some reasons